src/main/resources/processes/TO.bpmn

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The change business area functionality requires the stage on this diagram to support transferring it to the different business area, on the `TO.bpmn` it's a gateway labelled `Transfer Case?`